ras_party/controllers/account_controller.py

```diff
@@ -65,11 +65,11 @@ def post_respondent(party, tran, session):
 
     iac = request_iac(party['enrolmentCode'])
     if not iac.get('active'):
-        raise RasError("Enrolment code is not active.", status=400)
+        raise ClientError("Enrolment code is not active.", status=400)
 
     existing = query_respondent_by_email(party['emailAddress'], session)
     if existing:
-        raise RasError("Email address already exists.", status=400, party_uuid=party.get('party_uuid', None))
+        raise ClientError("Email address already exists.", status=400, party_uuid=party.get('party_uuid', None))
```

Comment:
Although `ClientError` persists the kwargs in a property, these aren't used when the exception is caught and the log line printed i.e. extra arguments like `party_uuid` won't be output in the logs.